modules/microshift-custom-ca-cert-cleaning.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+//
+// * microshift_security_compliance/microshift-custom-ca.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-custom-ca-certificates-cleaning_{context}"]
+= Cleaning and recreating the custom certificates
+
+To stop the {microshift-short} services, clean the {microshift-short} certificates and recreate the {microshift-short} certificates, use the following steps.
```

Comment:
What does "clean" mean? Are we removing previous certificates and their data? Be good say what and why and not use jargon.

Reply:
Yes. Okay.
changing 'clean' with 'clean up'.